npm/angular/src/mount.ts

```diff
@@ -235,6 +234,20 @@ function setupComponent<T> (
     })
   }
 
+  if (component.ngOnChanges && config.componentProperties) {
```

Comment:
From the contract it looks like `componentProperties` could be an empty object. Do we need to exclude that case from firing `ngOnChanges`?